plugin-k8s/src/cgroup_v2.rs

```diff
@@ -40,55 +59,103 @@ pub fn is_accessible_dir(path: &Path) -> bool {
     path.is_dir()
 }
 
-
 /// Returns a Vector of CgroupV2MetricFile associated to pods availables under a given directory.
```

Comment:
This was the original comment before it was fucked up by I don't know what

```suggestion
/// Returns a Vector of CgroupV2MetricFile associated to pods available under a given directory.
```